clean/README.md

```diff
@@ -0,0 +1,36 @@
+# pyATS | Clean Examples
+
+This repository contains usage examples for the pyATS clean library. These are
+not working examples, but rather:
+
+- showcase **the various features and functionalities** of the pyATS clean
+  framework
+
+- demonstrate **how to use the pyATS clean libraries** 
+
+## Getting Started
+
+```bash
+
+# 1. make sure pyATS is installed (including the libraries)
+bash$ pip install pyats[full]
```

Comment:
add the internal version too: pip install ats[full]

Reply:
added